src/sentry/replays/lib/summarize.py

```diff
@@ -271,8 +271,8 @@ def as_log_message(event: dict[str, Any]) -> str | None:
             case EventType.RAGE_CLICK:
                 message = event["data"]["payload"]["message"]
                 return f"User rage clicked on {message} but the triggered action was slow to complete at {timestamp}"
-            case EventType.NAVIGATION:
-                to = event["data"]["payload"]["data"]["to"]
```

Comment:
note that we lose the navigation type (e.g. push, reload, back/forward), but I think that's ok (though it might be useful for `reload`)